commercetools-models/src/main/java/io/sphere/sdk/products/search/ProductCategoriesIdTermFacetSearchModel.java

```diff
@@ -24,6 +24,9 @@
     @Override
     ProductCategoriesIdTermFacetSearchModel<T> withAlias(final String alias);
 
+    @Override
+    ProductCategoriesIdTermFacetSearchModel<T> countingProducts(final Boolean isCountingProducts);
```

Comment:
Thats true but the property of the class is called `isSomething` of `hasSomething` for Boolean properties. Dropping this breaks the symmetry `with<property-name>`. 
Its a matter of taste.